### ISSUE TITLE
Always got a warning when use ParticleContainer

### ISSUE THREAD

**Issue by finscn** [pixijs/pixi.js#3110]
The warning is :

```
[.Offscreen-For-WebGL-0x7fa497951a00]RENDER WARNING: there is no texture bound to the unit 0
```

Anything else is OK .

**Comment by englercj** [comment]
Looks like this reproduces on latest dev:

**Comment by finscn** [comment]
Yes . they are look the same.

**Comment by finscn** [comment]
@englercj  @ivanpopelyshev  , in the last version of PIXI , this issue is disappear.
Which  PR  fixed  this issue ?

**Comment by themoonrat** [comment]
https://github.com/pixijs/pixi.js/releases contains a list of commits for each release

**Comment by finscn** [comment]
I've not found which release fix #3110
